src/Akka.Bootstrap.Docker.Tests/DockerBootstrapSpecs.cs

```diff
@@ -19,18 +20,20 @@ namespace Akka.Bootstrap.Docker.Tests
     public class DockerBootstrapSpecs
     {
         [Theory]
-        [InlineData("akka.tcp://MySys@localhost:9140")]
-        [InlineData("akka.tcp://MySys@localhost:9140, akka.tcp://MySys@localhost:9141")]
-        [InlineData("akka.tcp://MySys@localhost:9140, akka.tcp://MySys@localhost:9141, akka.tcp://MySys@localhost:9142")]
+        [InlineData("[\"akka.tcp://MySys@localhost:9140\"]")]
```

Comment:
No, can't require brackets here - breaks backwards compat.

Reply:
Sorry for not being clear on this earlier - miscommunication on my part over Gitter